kv/kvnemesis: TestKVNemesisSingleNode failed

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://tanzanite.cluster.engflow.com/invocations/default/efc8a553-d34b-4b19-ae9c-5efa8025afc8?testReportRun=7&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZuZW1lc2lzOmt2bmVtZXNpc190ZXN0) on master @ [f0116ea373a2b87155e7f0264df4f783ce177360](https://github.com/cockroachdb/cockroach/commits/f0116ea373a2b87155e7f0264df4f783ce177360):

Fatal error:

```
panic: test timed out after 4m57s
running tests:
	TestKVNemesisSingleNode (4m54s)

```
Stack: 

```
goroutine 15825 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x385
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestKVNemesisSingleNode2322171947
    test_log_scope.go:81: use -show-logs to present logs inline
    kvnemesis_test.go:328: seed: 8204960555716622799
    kvnemesis_test.go:235: kvnemesis logging to /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/testlogs/pkg/kv/kvnemesis/kvnemesis_test/run_7_of_30/test.outputs/kvnemesis4077082983
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=7</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-36846

--- a/pkg/kv/kvnemesis/BUILD.bazel
+++ b/pkg/kv/kvnemesis/BUILD.bazel
@@ -72,7 +72,7 @@ go_test(
     ],
     data = glob(["testdata/**"]),
     embed = [":kvnemesis"],
-    exec_properties = {"test.Pool": "large"},
+    exec_properties = {"test.Pool": "heavy"},
     deps = [
         "//pkg/base",
         "//pkg/config/zonepb",

